syntax = "proto3";

package protobuff;

message ResponseTypePb {
    ResponseSatusEnum statusType = 1;
}

enum ResponseSatusEnum {
    UNKNOWN_RESPONSE_STATUS = 0;
    SUCCESS = 1;
    FAILED = 2;
    ERROR = 3;
    USER_EXIST = 4;
    USER_NOT_REGISTRED = 5;
    UNEXP_ERROR = 6;
    INVALID_CREDENTIALS = 7;
}
